tsconfig.cjs.json

```diff
@@ -4,5 +4,8 @@
     "outDir": "dist/cjs",
     "module": "CommonJS",
     "target": "ES5",
+    "downlevelIteration": true,
+    "importHelpers": true,
+    "lib": ["dom", "es5", "es2015.collection", "es2015.iterable", "es2019"]
```

Comment:
I'm afraid it is, otherwise we get:

```
yarn run v1.22.10
$ yarn build:clean && yarn typescript:build
$ rimraf dist
$ tsc && tsc --build tsconfig.cjs.json && tsc --build tsconfig.cli.json && chmod +x dist/cli/index.js
src/cli/main.ts:37:14 - error TS2339: Property 'flatMap' does not exist on type 'TaskGroupResult[][]'.

37             .flatMap((taskGroupResults) => taskGroupResults)
                ~~~~~~~

src/cli/main.ts:37:23 - error TS7006: Parameter 'taskGroupResults' implicitly has an 'any' type.

37             .flatMap((taskGroupResults) => taskGroupResults)
                         ~~~~~~~~~~~~~~~~

src/cli/main.ts:38:21 - error TS7031: Binding element 'groupId' implicitly has an 'any' type.

38             .map(({ groupId, map }) => [groupId, convertToTaskValueMap(map)] as [string, Results])
                       ~~~~~~~

src/cli/main.ts:38:30 - error TS7031: Binding element 'map' implicitly has an 'any' type.

38             .map(({ groupId, map }) => [groupId, convertToTaskValueMap(map)] as [string, Results])
                                ~~~

src/cli/main.ts:66:48 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'Results'.

66       const finalResults = performCalculations(result);
                                                  ~~~~~~


Found 5 errors.

error Command failed with exit code 1.
```

Reply:
Re: including or not `es5`, I'd argue "explicit is better than implicit" in this case. What do you think?

Reply:
Beaut.